stats_test.go

```diff
@@ -70,11 +70,11 @@ func TestUDPExportError(t *testing.T) {
 	}
 }
 
-func TestUDSExportError(t *testing.T) {
+func TestSubmitMetricError(t *testing.T) {
 	var expected error
 
 	exporter, err := testExporter(Options{
-		StatsAddr: "unix:///invalid.socket", // Ideally we wwouln't hit the filesystem.
+		StatsAddr: "unix:///dummy.socket",
```

Comment:
If this is a problem because of name clashing, dummy also doesn't sound that specific, why not go full on verbose `opencensus-go-exporter-datadog.test.socket` ?

Reply:
I changed to `dummy`, because we don't care anymore if it's valid or not in this test since we are testing this error instead (there is no issue with naming clash): https://github.com/DataDog/opencensus-go-exporter-datadog/blob/fe3fae706db1cdb6dd6c97b838c57637c15cfbbc/stats.go#L107

Reply:
Got it. Resolving.